### ISSUE TITLE
Installing Moya 15.0.0-alpha.1 via SPM results in Error

### ISSUE THREAD

**Issue by edulpn** [Moya/Moya#2109]
I'm currently using latest `Moya 14` installed via SPM on `Xcode 12.2`:
```swift
.package(
            url: "https://github.com/Moya/Moya.git",
            from: "14.0.0"
        )
```

I want to try `CombineMoya`, so when I bump the dependency on `Package.swift` I get the following error on Xcode:
![image](https://user-images.githubusercontent.com/4682317/100528400-2687a000-31bb-11eb-9c04-6b7ea8f22d90.png)

I'm using the following on my `Package.swift` to point to `15.0.0-alpha.1`:
```swift
.package(
            url: "https://github.com/Moya/Moya.git",
            .exact("15.0.0-alpha.1")
        )
```

Everything works fine before bumping `Moya` and I already tried on a clean project, so I'm pretty confident it's not another dependency issue.

**Comment by edulpn** [comment]
I also tried:
```swift
.package(url: "https://github.com/Moya/Moya.git",
                 .revision("e5a28fb62dd5ff4e17b7025643366550044a40b0") // commit hash for the 15.0.0-alpha.1 release
        )
```

```swift
.package(url: "https://github.com/Moya/Moya.git",
                 .exact(Version(15, 0, 0, prereleaseIdentifiers: ["alpha", "1"]))
        )
```

Still no luck

**Comment by edulpn** [comment]
@sunshinejr I found some information on the Combine mega thread #1928, but I'm still not able to make work. Have any ideas?

**Comment by Pomme2Poule** [comment]
Hey! Have you tried adding the package via SPM, using the Xcode interface instead of the package file? I made it work by selecting the commit hash just like you did. But I just let Xcode do the configuration for me.

**Comment by edulpn** [comment]
No, I haven't. In my past experiences Xcode was crap at managing packages
so I haven't considered that. Thanks!
